hw_37/src/App.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './App.css';
+import { Button } from './ui/button/Button';
+import { Header } from './features/header/Header';
+import { Description } from './ui/description/Description';
+import { Login } from './ui/form/Login';
+import { Registration } from './ui/form/Registration';
+import { WelcomeTemplate } from './templates/welcome/WelomeTemplate';
+import { ContentMainPage } from './pages/content-main-page/ContentMainPage';
+
+function App() {
+  return (
+    <div className="app">
+      <nav>
+        <div className="burger-btn">
+          <span></span>
+        </div>
+      </nav>
+      <div className="container">
+        <div className="main-page">
+          <Header>
+            Make your Blog{' '}
+            <span>
```

Comment:
It's OK to place this content inside Header feature. `children` prop usage is very limited for feature-components because UI incapsulated inside feature